build_tools/buildkite/cmake/linux/x86-64/benchmark.yml

```diff
@@ -0,0 +1,52 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+steps:
+  - label: "Build"
+    commands:
+      - "docker run --user=$(id -u):$(id -g) --volume=\\${HOME?}:\\${HOME?} --volume=/etc/passwd:/etc/passwd:ro --volume=/etc/group:/etc/group:ro --volume=\\$PWD:\\$IREE_DOCKER_WORKDIR --workdir=\\$IREE_DOCKER_WORKDIR --rm gcr.io/iree-oss/frontends@sha256:e0a5d9a71cc7f087f4caaea0cc9b9a86ab4765b217970772780e446b4fdc65b0 build_tools/cmake/build_linux_benchmark.sh"
+      - "tar --exclude='*.tar.gz' --exclude='*.tgz' --exclude='*.mlir' --exclude='*.tflite' -czvf benchmark-suites-${BUILDKITE_BUILD_NUMBER}.tgz build-host/benchmark_suites"
+      - "find build-host/benchmark_suites -name '*.mlir' | tar -czvf source-mlir-models-${BUILDKITE_BUILD_NUMBER}.tgz -T -"
+      - "tar -czvf iree-linux-x86-64-tools-${BUILDKITE_BUILD_NUMBER}.tgz build-linux-x86-64/iree/tools/iree-benchmark-module build-linux-x86-64/iree/tools/build_config.txt"
+    if: "build.pull_request.id == null || (build.pull_request.labels includes 'buildkite:benchmark-x86')"
+    agents:
+      - "queue=build"
+    env:
+      IREE_DOCKER_WORKDIR: "/usr/src/github/iree"
+    artifact_paths:
+      - "benchmark-suites-${BUILDKITE_BUILD_NUMBER}.tgz"
+      - "source-mlir-models-${BUILDKITE_BUILD_NUMBER}.tgz"
+      - "iree-linux-x86-64-tools-${BUILDKITE_BUILD_NUMBER}.tgz"
+
+  - wait
+
+  - label: "Benchmark on Intel Cascade Lake CPU (GCP-c2-standard-16)"
+    commands:
+      - "git clean -fdx"
+      - "buildkite-agent artifact download --step Build benchmark-suites-${BUILDKITE_BUILD_NUMBER}.tgz ./"
+      - "buildkite-agent artifact download --step Build iree-linux-x86-64-tools-${BUILDKITE_BUILD_NUMBER}.tgz ./"
+      - "tar -xzvf benchmark-suites-${BUILDKITE_BUILD_NUMBER}.tgz"
+      - "tar -xzvf iree-linux-x86-64-tools-${BUILDKITE_BUILD_NUMBER}.tgz"
+      - "python3 build_tools/benchmarks/run_benchmarks_on_linux.py --device_model=GCP-c2-standard-16 --cpu_uarch=CascadeLake --normal_benchmark_tool_dir=build-linux-x86-64/iree/tools/ -o benchmark-results-gcp-cpu-${BUILDKITE_BUILD_NUMBER}.json --verbose build-host/"
+    if: "build.pull_request.id == null || (build.pull_request.labels includes 'buildkite:benchmark-x86')"
+    agents:
+      - "gcp:machine-type=c2-standard-16"
+      - "queue=benchmark-x86"
+    artifact_paths:
+      - "benchmark-results-gcp-cpu-${BUILDKITE_BUILD_NUMBER}.json"
+    timeout_in_minutes: "60"
```

Comment:
Can we shorten this timeout? https://buildkite.com/iree/jerry-benchmark-pipeline/builds/38 looks like it only took 3 minutes, and if the pipeline stalls for some reason then a shorter timeout would catch that faster.

Reply:
Shorten to 10mins.